crates/cli/tests/cli_run.rs

```diff
@@ -1030,7 +999,7 @@ mod cli_run {
             &[],
             &[],
             &[],
-            "27101\n",
+            "6239\n",
```

Comment:
This now uses the new test-file.txt which should not change in size in the future.